mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -21,6 +21,73 @@ message CreateRegisteredModel {
   }
 }
 
+message DeleteRegisteredModel {
+  // Required. Full name of the model.
+  optional string full_name = 1 [(validate_required) = true];
+  // Optional. Force deletion even if the registered model is not empty. Default is false.
+  optional bool force = 2;
+
+  message Response {}
+}
+
+message GetRegisteredModel {
+
+  // Required. Full name of the Model.
+  optional string full_name = 1 [(validate_required) = true];
+
+  message Response {
+    optional RegisteredModelInfo registered_model_info = 1 [(json_inline) = true];
+  }
+}
+
+message CreateModelVersion {
+  optional ModelVersionInfo model_version_info = 1 [(json_inline) = true];
+
+  message Response {
+    optional ModelVersionInfo model_version_info = 1 [(json_inline) = true];
+  }
+}
+
+message UpdateRegisteredModel {
```

Comment:
Can we organize these messaged by RegisteredModel operations and model version operations?